SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJobWebRunner/BrokerProfileScanSubJobWebRunner.swift

```diff
@@ -23,39 +23,45 @@ import UserScript
 import Common
 import os.log
 
-final class ScanJob: DataBrokerJob {
-    typealias ReturnValue = [ExtractedProfile]
-    typealias InputValue = Void
+public protocol BrokerProfileScanSubJobWebRunning {
+    func scan(_ profileQuery: BrokerProfileQueryData,
+              showWebView: Bool,
+              shouldRunNextStep: @escaping () -> Bool) async throws -> [ExtractedProfile]
+}
+
+public final class BrokerProfileScanSubJobWebRunner: SubJobWebRunning, BrokerProfileScanSubJobWebRunning {
+    public typealias ReturnValue = [ExtractedProfile]
+    public typealias InputValue = Void
```

Comment:
I know they were there already, but I don't like having these generically named input and return values, they seem to just serve to obscure things

Reply:
Yeah I agree here, it feels like a bit of a hack to make this protocol work on two totally different types of jobs, if that makes sense. Would like to find a better approach tbh, not sure what that looks like yet though.

Reply:
This isn't helpful because I can't remember the details, but I think I tried this at one point and did find something that worked, but at the time I was afraid to change anything 😅